libbeat/scripts/Makefile

```diff
@@ -173,19 +173,19 @@ prepare-tests:
 .PHONY: unit-tests
 unit-tests: ## @testing Runs the unit tests with coverage.  Race is not enabled for unit tests because tests run much slower.
 unit-tests: prepare-tests
-	go test -i ${GOPACKAGES}
+	go test ${GOPACKAGES}
 	$(COVERAGE_TOOL) $(RACE) -coverprofile=${COVERAGE_DIR}/unit.cov  ${GOPACKAGES}
 
 .PHONY: unit
 unit: ## @testing Runs the unit tests without coverage reports.
-	go test -i ${GOPACKAGES}
+	go test ${GOPACKAGES}
```

Comment:
this would actually duplicate test runs, to get rid of the `-i` flag this line would need to be removed entirely, as tests are run in the next one. Same applies to the whole change